src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -131,6 +131,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('list_enabled', help='Indicates whether this item shows in list operation results.', arg_type=get_three_state_flag())
         c.argument('read_enabled', help='Indicates whether read operation is allowed.', arg_type=get_three_state_flag())
         c.argument('write_enabled', help='Indicates whether write or delete operation is allowed.', arg_type=get_three_state_flag())
+        c.argument('teleport_enabled', help='Indicates whether Teleport feature is enabled on the repository.', arg_type=get_three_state_flag())
```

Comment:
Is it possible to mark the parameter as preview? If not, can the help text somehow indicate it's a preview feature?

Reply:
add the flag is_preview=True,

Reply:
Specify default falue

Reply:
the default value should not be false. We don't want to set it to `false` when the user is changing other properties.